03LessonThree/LessonThree.js

```diff
@@ -2,33 +2,37 @@ module.exports = function() {
 
   this.buildArray = function (first, second, third) {
     // return array with the items first, second, and third in an array.
-
+    return [first, second, third];
   }
 
   this.returnThirdItem = function (arr) {
     // should return the third item in the array
-
+    return arr[2];
   }
 
   this.setFirstItem = function (arr, newFirstItem) {
     // should set the first item in the array with newFirstItem
-
+    arr[0]= newFirstItem;
+    return arr;
   }
 
   this.returnCenterItem = function(fiveByFiveArray) {
     // returns the "center" item in a 5 x 5 array
     // ex. 3 x 3 array [[1, 2, 3], [4, 5, 6], [7, 8, 9]] the center item is 5
-
+    
+    // fiveByFiveArray = Math.ceil(fiveByFiveArray.length/2)
```

Comment:
I like what you were trying to do here! Finding the center dynamically instead of hardcoding the indices would be a good challenge.